lms/static/js/courseware/credit_progress.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function() {
+    $('.detail-collapse').click(function(){
```

Comment:
Can you please update to us .on('click', to follow conventions throughout the rest of the site? Also, it will be clearer if you pass values through the function and use those instead of this. e.g.
```
$('.detail-collapse').on('click', function( i, el ) {
    $(el).find.......
```